client/index.html

```diff
@@ -2,24 +2,32 @@
 <html>
 
 <head>
-	<meta charset="UTF-8">
-	<title>CheatSheet</title>
+    <meta charset="UTF-8">
+    <title>CheatSheet</title>
```

Comment:
Aight, one thing I am picky on is tabs > spaces.

Reply:
yeah, didn't mean to run my formatter on anything. should be fixed now that I pulled in ben's changes